library/src/main/java/br/com/goncalves/pugnotification/notification/Load.java

```diff
@@ -22,14 +22,14 @@
 import br.com.goncalves.pugnotification.pendingintent.DismissPendingIntentBroadCast;
 
 public class Load {
-    private static final String TAG = "Pugnotification.Load";
     private NotificationCompat.Builder builder;
     private String title;
     private String message;
     private Spanned messageSpanned;
     private String tag;
     private int notificationId;
     private int smallIcon;
+    private boolean useSpanForCustomNotification;
```

Comment:
Shouldn't this be initialized to `true` so we don't accidentally change behaviour of notifications that already use spanned text?